code/datums/status_effects/debuffs.dm

```diff
@@ -287,6 +288,84 @@
 	owner.underlays -= marked_underlay //if this is being called, we should have an owner at this point.
 	..()
 
+/datum/status_effect/eldritch
+	duration = 15 SECONDS
+	status_type = STATUS_EFFECT_REPLACE
+	alert_type = null
+	var/mutable_appearance/marked_underlay
+	var/effect_sprite = ""
```

Comment:
Document the vars.

Reply:
Please do not resolve unaddressed requests.

Reply:
i thought i documented these... nevermind